package/yast2-apparmor.changes

```diff
@@ -1,3 +1,12 @@
+-------------------------------------------------------------------
+Wed Jul 17 07:25:37 UTC 2019 - David Diaz <dgonzalez@suse.com>
+
+- Avoid crashes due to dead code related to notify settings, which
+  were removed some time ago (bsc#1139852).
+- Removes a Perl script to generate reports that is broken
+  and is not being used from a long time ago.
```

Comment:
NP: I would use "since" instead of "from"

Reply:
or "is not used for a long time"